lib/libutee/tee_api_panic.c

```diff
@@ -22,6 +22,9 @@
 void TEE_Panic(TEE_Result panicCode)
 {
 	_utee_panic(panicCode);
+#ifdef __COVERITY__
+	__coverity_panic__();
+#endif
```

Comment:
Normally I would agree, but in this case I think this is the only place where this will be needed. If we need any more of these special Coverity function I'll add a `<coverity.h>` etc. Is that OK?

Reply:
Yeah fine with me.